docker-compose.yaml

```diff
@@ -56,6 +56,7 @@ services:
       - PUBSUB_CCA_TOPIC_ID=ocs-cca
       - PUBSUB_CCA_SUBSCRIPTION_ID=ocsgw-cca-sub
       - PUBSUB_ACTIVATE_SUBSCRIPTION_ID=ocsgw-activate-sub
+      - OCS_DATASOURCE_TYPE=Proxy
```

Comment:
Does this mean we will not involve OCS in Prime during AT?

Reply:
It will use the same setup as we use in dev/prod.  CRR - Init and Terminate will go to Prime/OCS. Update will be local loopback but allse sent to Prime/OCS.

Reply:
Oh, sorry.  This is Proxy, not Local!